testsuite/tests/performance/conftest.py

```diff
@@ -115,3 +115,13 @@ def promoted_services(services, production_gateway):
         svc.proxy.list().promote(version=version)
     production_gateway.reload()
     return services
+
+
+@pytest.fixture(scope="module")
+def prod_client(request):
+    """Production client for performance tests"""
+    def _client(app):
```

Comment:
Sparse docstring (purpose of the fixture?)

Reply:
@mganisin Added description